ash/src/extensions/khr/create_render_pass2.rs

```diff
@@ -1,26 +1,11 @@
-//! <https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_KHR_create_render_pass2.html>
+//! <https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_KHR_create_renderpass2.html>
```

Comment:
Nice catch! I generated these from filenames, if you can please rename this one.

Reply:
Done.